modules/channel-web/src/views/lite/utils.tsx

```diff
@@ -66,6 +66,24 @@ export const initializeAnalytics = () => {
   }
 }
 
+export const trackMessage = (direction: 'sent' | 'received') => {
+  if (window.SEND_USAGE_STATS) {
+    try {
+      ReactGA.event({ category: 'Interactions', action: 'message ' + direction })
```

Comment:
I know that's a very picky comment... we're using string interpolation pretty much everywhere can you please change this for `message ${direction}`